cypress/integration/apis/animalandveterinary/event/endpoint_pages.spec.js

```diff
@@ -0,0 +1,47 @@
+/// <reference types="cypress" />
+import EndpointPagesTestHelper from "../../../../support/endpoint_test_helper";
+import {loadAndAcceptDisclaimer} from "../../../../support/index";
+
+const URL = '/apis/animalandveterinary/event/';
+
+context('ADAE dataset pages', () => {
+    var helper;
+
+    beforeEach(() => {
+        loadAndAcceptDisclaimer(URL);
+        helper = new EndpointPagesTestHelper(URL, '/animalandveterinary/event.json');
```

Comment:
The generic helpers are excellent and helping a lot, but we're missing a number of link validations as a result of making the endpoint page checks completely generic. The overview pages generally have variable links based on the needs of that specific endpoint, and we should attempt to capture those in tests.